deploy/deploy.ts

```diff
@@ -92,11 +109,120 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
 
   dim(`deployer: ${admin}`);
 
-  cyan(`\nDeploying ATokenYieldSourceProxyFactory...`);
-  const aTokenYieldSourceProxyFactoryResult = await deploy('ATokenYieldSourceProxyFactory', {
+  cyan(`\nDeploying ATokenYieldSource...`);
+  const aTokenYieldSourceResult = await deploy('ATokenYieldSource', {
     from: deployer,
+    skipIfAlreadyDeployed: true
   });
-  displayResult('ATokenYieldSourceProxyFactory', aTokenYieldSourceProxyFactoryResult);
+  
+  displayResult('ATokenYieldSource', aTokenYieldSourceResult);
+
+  let aTokenYieldSourceContract
+  if(aTokenYieldSourceResult.address){
+    aTokenYieldSourceContract = await ethers.getContractAt("ATokenYieldSource", aTokenYieldSourceResult.address)
+    dim(`calling mockInitialize()`)
+    //await aTokenYieldSourceContract.mockInitialize()
+    green(`mockInitialize success`)
```

Comment:
✂️ Looks like dead code / comments

Reply:
This is needed! but instead we should check if the intialization has already happened or not to make this script re-runnable.

Reply:
Instead checking `newlyDeployed` flag